server/routes/static.py

```diff
@@ -31,6 +32,8 @@
 @bp.route('/')
 def homepage():
   # Return old homepage if hiding revamp changes
+  if request.url == 'https://unsdg.datacommons.org/':
```

Comment:
Do we want to include "http://" as well as without trailing "/" ?

Reply:
from local try, looks like they are in it..

Reply:
sg